### DISCUSSION TITLE
Clarification in documentation

### DISCUSSION THREAD

**Discussion by kiadso** [chipzoller/hugo-clarity#138] [Q&A]
When I go to add a thumbnail to a post, the path starts at "content", not at the the post's directory, so I had to add some directories to the thumbnail reference to get it to work. The documentation doesn't specify this. Here's what I used that worked:

thumbnail: "post/post-slug/images/thumbnail.jpg"

Thank you for this wonderful theme and great documentation!

**Comment by chipzoller** [comment]
I thought that it defaults to the /static dir as a base. Anything specified from there forward is what goes into the `thumbnail` parameter. So you specifying post/ is strange. Maybe this was a Hugo change? @onweru does this sound right?

**Comment by onweru** [comment]
@kiadso, 

We are following [Hugo's static files management](https://gohugo.io/content-management/static-files/) lead. @chipzoller is right that the `static` dir is used as a base.

If your images are stored in `/static/images` dir, what you currently has as as `post/post-slug/images/thumbnail.jpg` ought to be `/images/thumbnail.jpg`
